tests/test_images.py

```diff
@@ -214,6 +214,66 @@ def test_image_extraction(src, page_index, image_key, expected):
     assert image_similarity(BytesIO(actual_image.data), expected) >= 0.99
 
 
+@pytest.mark.enable_socket()
+def test_onlyoffice_standard_images_extraction():
+    reader = PdfReader(
+        BytesIO(get_data_from_url(name="iss2613-onlyoffice-standardImages.pdf"))
```

Comment:
as you are having url/name defined locally I recommend to do the same for the pdf and not change the yaml file above